composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "bc135d31f47171de9a1a55745e62b79d",
+    "content-hash": "3aa3c5f1cf0ab18b0a901d9c0d0489bc",
```

Comment:
composer was giving a notice that the composer.lock is not up to date with composer.json - my fault; I didn't update it after moving the features and config_update modules from `require` into `require-dev`